x-pack/plugins/aiops/public/components/change_point_detection/change_point_detection_page.tsx

```diff
@@ -129,7 +189,7 @@ export const ChangePointDetectionPage: FC = () => {
               <p>
                 <FormattedMessage
                   id="xpack.aiops.changePointDetection.noChangePointsFoundMessage"
-                  defaultMessage="Try to extend the time range or update the query"
+                  defaultMessage="Detect statistically significant change points such as dips, spikes, and distribution changes in a metric. Select a metric and set a time range to start detecting change points in your data"
```

Comment:
```suggestion
                  defaultMessage="Detect statistically significant change points such as dips, spikes, and distribution changes in a metric. Select a metric and set a time range to start detecting change points in your data."
```

Reply:
Fixed in https://github.com/elastic/kibana/pull/150308/commits/26a638820227f69f2ba519ad6127e84a43dbcfcc